README.md

```diff
@@ -128,15 +128,12 @@ $ brew install ponyc
 
 ## Windows using ZIP (via Bintray)
 
-For Windows, the `master` and `release` branches are packaged and available on Bintray ([pony-language/ponyc-win](https://bintray.com/pony-language/ponyc-win)):
+Windows users will need to install:
 
-```powershell
-Invoke-WebRequest -Uri https://dl.bintray.com/pony-language/ponyc-win/ponyc-VERSION.zip -UseBasicParsing -OutFile ponyc-VERSION.zip
-7z x .\ponyc-VERSION.zip
-.\ponyc-VERSION\ponyc\bin\ponyc.exe --version
-```
+- Visual Studio 2015 (or the Visual C++ Build Tools 2015) They can be downloaded from Microsoft: ([Visual Studio](https://www.visualstudio.com/vs/community/), [Visual C++ Build Tools 2015](http://landinghub.visualstudio.com/visual-cpp-build-tools)
+- Windows 10 SDK in order to build programs with ponyc. [SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)).
```

Comment:
And how about:
```
- Windows 10 SDK (available [here](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)).
```

Note my list convention of "X, and Y." is a vestige of working for lawyers writing technical stuff: what matters here is just wording.